packages/dds/task-manager/src/interfaces.ts

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ISharedObject, ISharedObjectEvents } from "@fluidframework/shared-object-base";
+
+export interface ITaskManagerEvents extends ISharedObjectEvents {
+    (event: "assigned" | "lost", listener: (taskId: string) => void);
```

Comment:
Where do the docstrings for the events go?

Reply:
Good question - I don't think we have precedent in our repo that I've seen?  I see that [ConsensusOrderedCollection](https://fluidframework.com/apis/ordered-collection/iconsensusorderedcollectionevents/) tried to document its events but it doesn't show up in the outputted [documentation](https://fluidframework.com/apis/ordered-collection/iconsensusorderedcollectionevents/)